test/metabase/search/scoring_test.clj

```diff
@@ -21,9 +21,22 @@
     (is (= []
            (scoring/tokenize " \t\n\t ")))
     (is (= []
-           (scoring/tokenize "")))
-    (is (thrown-with-msg? Exception #"does not match schema"
-                          (scoring/tokenize nil)))))
```

Comment:
Note for reviewers: I removed the schema check on tokenize because it was slowing down the search (by approx 1s if the search returns 1000 results)

Reply:
Would it make sense to keep this explanation in the code as a comment for future generations?

Reply:
Yes, added. And I realised the schema check was only hurting performance for the return value schema, not the arguments. See the update in this commit
https://github.com/metabase/metabase/pull/24171/commits/e6dcb015ea69f209a98004cc1a4dcaaabef4a87c